backend/src/main/resources/application.conf

```diff
@@ -41,6 +41,10 @@ services {
     name = "people.lunatech.com:80"
     api-key = ${PEOPLE_API_KEY}
   }
+  skillmatrix {
+    name = "techmatrix.lunatech.com:80"
+    api-key = ${TECHMATRIX_API_KEY}
```

Comment:
I am not sure how the application is deployed but we need to provide TECHMATRIX_API_KEY env variable.

Reply:
Yes, it's not a problem, CleverCloud let us defined env vars, PEOPLE_API_KEY is the same